src/cursor.js

```diff
@@ -3,78 +3,86 @@ import convertToNative from './convert_to_native';
 import Immutable from 'immutable';
 import putOnChan from './put_on_chan';
 
-const hasSameValue = function (valueA, cursorB) {
+function hasSameValue(valueA, cursorB) {
   return isEqual(valueA, Immutable.fromJS(cursorB.deref()));
-};
+}
 
-const isEqual = function (valueA, valueB) {
+function isEqual(valueA, valueB) {
```

Comment:
You don't have to pull Ramda in. You just need to add an equals() method to the cursor prototype and have it call isEqual. Ramda looks for an equals() method on things you pass in to equals.